orca-clouddriver/src/main/java/com/netflix/spinnaker/orca/clouddriver/tasks/cluster/DetermineRollbackCandidatesTask.java

```diff
@@ -79,6 +80,9 @@ public class DetermineRollbackCandidatesTask implements CloudProviderAware, Retr
   private static final TypeReference<List<ServerGroup>> listOfServerGroupsTypeReference =
       new TypeReference<>() {};
 
+  @Value("${rollback.timeout.enabled:false}")
```

Comment:
The name of this property could be confusing to users and developers. It looks like the timeout functionality would be disabled completely if the flag is set to false or not set at all. Would it make sense to rename it to something like `rollback.dynamic-timeout.enabled` ?